pyvista/plotting/mapper.py

```diff
@@ -869,7 +883,7 @@ def __init__(self, theme=None):
     @property
     def dataset(self):
         """Return or set the dataset assigned to this mapper."""
-        return self.GetInputAsDataSet()
+        return self.GetDataSetInput()
```

Comment:
Why is this changing? 

This has consequences in other places and `GetInputAsDataSet()` should be used

Reply:
Oh wait. I got mixed up. This is specifically for the volume mapper.

Reply:
This change is fine. I didn't realize this was on `_BaseVolumeMapper`. And I just noticed that `GetInputAsDataSet()` is not available on the volume mapper classes, thus requiring `GetDataSetInput()`

Reply:
Perhaps we should add a comment that `GetInputAsDataSet()` is not available on the volume mappers